File: Java/src/main/java/com/contidio/sdk/exceptions/InvalidOperationBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class InvalidOperationBackendException extends BackendException {
	public InvalidOperationBackendException(BackendErrorCode errorCode, String message) {
		super(errorCode, message, "InvalidOperationBackendException", HttpStatus.SC_BAD_REQUEST);
	}	
}

File: .NET Core/Exceptions/InvalidOperationBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class InvalidOperationBackendException : BackendException
    {
        public InvalidOperationBackendException(BackendErrorCode errorCode, string message)
            : base(errorCode, message, "InvalidOperationBackendException", HttpStatusCode.BadRequest)
        {
        }
    }
}
